pkg/controller/machine/controller.go

```diff
@@ -238,3 +246,16 @@ func (r *ReconcileMachine) isDeleteAllowed(machine *clusterv1.Machine) bool {
 	// own machine.
 	return node.UID != machine.Status.NodeRef.UID
 }
+
+func (r *ReconcileMachine) deleteNode(ctx context.Context, name string) error {
+	var node corev1.Node
+	if err := r.Client.Get(ctx, client.ObjectKey{Name: name}, &node); err != nil {
+		if apierrors.IsNotFound(err) {
+			klog.Infof("Node %q not found", name)
```

Comment:
Should we increase the verbosity for this error message?

Reply:
v(2)?